plugins/osdn/common.go

```diff
@@ -26,11 +26,11 @@ type PluginHooks interface {
 
 	SetupSDN(localSubnetCIDR, clusterNetworkCIDR, serviceNetworkCIDR string, mtu uint) (bool, error)
 
-	AddOFRules(nodeIP, nodeSubnetCIDR, localIP string) error
-	DelOFRules(nodeIP, localIP string) error
+	HostSubnetAdded(subnet *osapi.HostSubnet)
+	HostSubnetDeleted(subnet *osapi.HostSubnet)
 
-	AddServiceOFRules(netID uint, IP string, protocol kapi.Protocol, port int) error
-	DelServiceOFRules(netID uint, IP string, protocol kapi.Protocol, port int) error
+	ServiceAdded(service *kapi.Service, netID uint)
+	ServiceDeleted(service *kapi.Service, netID uint)
 
```

Comment:
AddHostSubnet, DeleteHostSubnet, AddService, DeleteService names will be better. These will denote intent instead of completed action names and also it will be inline with SetupSDN and UpdatePod (action+object).